### DISCUSSION TITLE
Possible to edit php.ini and enable ffi extension

### DISCUSSION THREAD

**Discussion by stayathomedad** [nextcloud/all-in-one#1124] [Questions] [answered]
Hi. Newbie here. I have AIO up and running behind Cloudflare tunnel and everything is awesome. However, on my old instance, I was using Storj app for external storage and would like to use it with AIO as well. I need to modify php.ini and enable ffi extension. Is there a reasonably simple way to do this? Thank you very much.

**Comment by szaimen** [comment, marked as answer]
I fear it is not possible to add the php extensions permanently. 
Also the app is not compatible with NC24? See https://apps.nextcloud.com/apps/storj

**Reply by stayathomedad** [reply]
okay thank you for your reply. much appreciated.
